pkg/service/record/record.go

```diff
@@ -79,49 +81,49 @@ func (r *recorder) CaptureTraffic(path string, proxyPort uint32, appCmd, appCont
 		loadedHooks.Stop(true)
 		return
 	default:
-		// start the BootProxy
-		ps = proxy.BootProxy(r.Logger, proxy.Option{Port: proxyPort}, appCmd, appContainer, 0, "", ports, loadedHooks, ctx, 0)
+		// Start the BootProxy
+		ps = proxy.BootProxy(r.Logger, proxy.Option{Port: params.ProxyPort}, params.AppCmd, params.AppContainer, 0, "", params.Ports, loadedHooks, ctx, 0)
 	}
 
-	//proxy fetches the destIp and destPort from the redirect proxy map
-	//Sending Proxy Ip & Port to the ebpf program
+	// Proxy fetches the destIp and destPort from the redirect proxy map
+	// Sending Proxy IP & Port to the ebpf program
 	if err := loadedHooks.SendProxyInfo(ps.IP4, ps.Port, ps.IP6); err != nil {
 		return
 	}
 
 	// Channels to communicate between different types of closing keploy
-	abortStopHooksInterrupt := make(chan bool) // channel to stop closing of keploy via interrupt
-	exitCmd := make(chan bool)                 // channel to exit this command
-	abortStopHooksForcefully := false          // boolen to stop closing of keploy via user app error
+	abortStopHooksInterrupt := make(chan bool) // Channel to stop closing of keploy via interrupt
+	exitCmd := make(chan bool)                 // Channel to exit this command
+	abortStopHooksForcefully := false          // Boolean to stop closing of keploy via user app error
 
 	select {
 	case <-stopper:
 		loadedHooks.Stop(true)
 		ps.StopProxyServer()
 		return
 	default:
-		// start user application
+		// Start user application
 		go func() {
 			stopApplication := false
-			if err := loadedHooks.LaunchUserApplication(appCmd, appContainer, appNetwork, Delay, buildDelay, false); err != nil {
+			if err := loadedHooks.LaunchUserApplication(params.AppCmd, params.AppContainer, params.AppNetwork, params.Delay, params.BuildDelay, false); err != nil {
 				switch err {
 				case hooks.ErrInterrupted:
 					r.Logger.Info("keploy terminated user application")
 					return
 				case hooks.ErrCommandError:
 				case hooks.ErrUnExpected:
-					r.Logger.Warn("user application terminated unexpectedly hence stopping keploy, please check application logs if this behaviour is not expected")
+					r.Logger.Warn("user application terminated unexpectedly hence stopping keploy, please check application logs if this behavior is not expected")
 				case hooks.ErrDockerError:
 					stopApplication = true
```

Comment:
unnecessary change

Reply:
behaviour spelling is fixed in the above change

Reply:
Both are acceptable spellings.

Reply:
Do you want the change reverted?

Reply:
yes can be reverted as someone might update to the same in near future.

Reply:
Reverted